src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colour getRandomColor() {
+        int index = new Random().nextInt(Colour.values().length);
```

Comment:
Directly accessing 'Colour.values()' inside a loop or method that is called multiple times can be inefficient because 'values()' creates a new array each time it is called. Consider storing the result of 'Colour.values()' in a constant field.

Reply:
Fixed